modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -137,14 +140,20 @@ ComputeMultiPlasticityStress::computeQpStress()
   bool ld_encountered = false;
   bool constraints_added = false;
 
-  // try a purely elastic step first
-  bool found_solution = elasticStep(_stress_old[_qp], _stress[_qp], _intnl_old[_qp], _intnl[_qp], _plastic_strain_old[_qp], _plastic_strain[_qp], _my_elasticity_tensor, _my_strain_increment, _yf[_qp], number_iterations, _Jacobian_mult[_qp]);
+  // plastic multipliers and cumulative plastic multipliers.
+  // Here they are used by quickStep below and are unimportant dummies
+  std::vector<Real> pm;
```

Comment:
made _dummy_pm, which is a member variable.  Also _cumulative_pm member variable, which avoids one extra "assign" in some cases